File: 0026-remove-duplicates-from-sorted-array/0026-remove-duplicates-from-sorted-array.java
class Solution {
    public int removeDuplicates(int[] nums) {

        int firstPointer = 0;
        for(int secondPointer = 0; secondPointer < nums.length;secondPointer++){
            if(nums[firstPointer] != nums[secondPointer]){
                firstPointer++;
                nums[firstPointer] = nums[secondPointer];
            }else{
                continue;
            }
        }

        return firstPointer+1;
        
    }
}

File: 0026-remove-duplicates-from-sorted-array/0026-remove-duplicates-from-sorted-array.py
class Solution:
    def removeDuplicates(self, nums: List[int]) -> int:

        n = len(nums)
        count = 0

        for i in range(n):
            if((i < n -1) and nums[i] == nums[i+1]):
                continue
            else:
                nums[count] = nums[i]
                count+=1
        return count
        